tests/test_build_html5.py

```diff
@@ -0,0 +1,327 @@
+# -*- coding: utf-8 -*-
+"""
+    test_build_html5
+    ~~~~~~~~~~~~~~~~
+
+    Test the HTML5 writer and check output against XPath.
+
+    This code is digest to reduce test running time.
+    Complete test code is here:
+
+    https://github.com/sphinx-doc/sphinx/pull/2805/files
+
+    :copyright: Copyright 2007-2017 by the Sphinx team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import os
+import re
+from itertools import cycle, chain
+
+from six import PY3
+
+from sphinx import __display_version__
+from util import remove_unicode_literals, strip_escseq
+from etree13 import ElementTree
```

Comment:
Unfortunately, `etree13` is removed at #3483. Could you merge the HEAD of master (I just merged the change into master now)?